tox.ini

```diff
@@ -0,0 +1,33 @@
+# -*- coding: utf-8 -*-
```

Comment:
We're normally not using Tox anymore. Usually Tox is not used enough by developers and thus the tox.ini ends up not being maintained properly.